File: common-core/src/main/java/com/ityu/bean/entity/system/Notice.java
package com.ityu.bean.entity.system;


import com.ityu.bean.entity.BaseEntity;
import lombok.Data;
import org.hibernate.annotations.Table;

import javax.persistence.Column;
import javax.persistence.Entity;

/**
 * Created  on 2018/4/2 0002.
 *
 * @author enilu
 */

@Entity(name = "t_sys_notice")
@Table(appliesTo = "t_sys_notice",comment = "通知")
@Data
public class Notice extends BaseEntity {
    @Column
    private String title;
    @Column
    private Integer type;
    @Column
    private String content;

}

File: common-core/src/main/java/com/ityu/common/bean/entity/system/Notice.kt
package com.ityu.common.bean.entity.system

import com.ityu.common.bean.entity.BaseEntity
import jakarta.persistence.Table
import org.springframework.data.jpa.domain.support.AuditingEntityListener
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.EntityListeners

/**
 * Created  on 2018/4/2 0002.
 * 系统通知，改表并没有真正用起来，可以根据自己实际需要做调整
 * @author enilu
 */
@Entity(name = "t_sys_notice")
@Table(name = "t_sys_notice", )
@EntityListeners(
    AuditingEntityListener::class
)
open class Notice : BaseEntity() {
    @Column(columnDefinition = "VARCHAR(64) COMMENT '通知标题'")
    open var title: String? = null

    @Column(columnDefinition = "INT COMMENT '通知类型'")
    open var type: Int? = null

    @Column(columnDefinition = "TEXT COMMENT '通知内容'")
    open var content: String? = null
}
